{"name": "用户手机登入", "status": "passed", "attachments": [{"name": "stdout", "source": "70f89a5a-3e9f-4700-8a31-da41186a407c-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "datas", "value": "{'username': 15000000002, 'password': 12345, 'expected': '无法使用提供的认证信息登录'}"}], "start": 1671357630426, "stop": 1671357630610, "uuid": "7c18b62c-83df-4a7d-9249-7ad982f26264", "historyId": "784ea7d33db8ebe1dcbc60f05601dd0a", "testCaseId": "7f5d9b461645abeb9a19f6ec1503a4c7", "fullName": "test_case.usercenter.test_user.TestUser#test_login", "labels": [{"name": "feature", "value": "用户中心模块"}, {"name": "story", "value": "用户登入"}, {"name": "parentSuite", "value": "test_case.usercenter"}, {"name": "suite", "value": "test_user"}, {"name": "subSuite", "value": "TestUser"}, {"name": "host", "value": "xiaoma"}, {"name": "thread", "value": "274080-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.usercenter.test_user"}]}